cypress/integration/block-text.js

```diff
@@ -27,6 +27,36 @@ if (Cypress.env('API') !== 'guillotina') {
         .should('have.attr', 'href')
         .and('include', 'https://google.com');
     });
+
+    it('As a site administrator I can add a new line to a text block using shift + enter', function() {
+      cy.visit('/');
+      cy.get('#toolbar-add').click();
+      cy.get('#toolbar-add-document').click();
+      cy.get('.documentFirstHeading > .public-DraftStyleDefault-block')
+        .type('This is a page')
+        .get('.documentFirstHeading span[data-text]')
+        .contains('This is a page');
+      //when I select a text-block and hit shift+enter or ctrl+enter
+      cy.get('.block.inner.text:nth-child(2n) .public-DraftEditor-content')
+        .type('First line;{shift}{enter}')
+        .type('Second line;{control}{enter}')
+        .type('Third line.')
+        .get('span[data-text]')
+        .contains(`First line;
+Second line;
+Third line.`);
+      //then the block should not be deselected
+      cy.get('.block.inner.text:nth-child(2n) .block.text.selected');
+      //then there still should not have <BR> tag
```

Comment:
I don't fully get what we are checking here. Why is the absence of a br tag something we check? If there is no way to visually check DraftJS for a BR-like tag, we shouldn't check anything at all.